### PR TITLE
feat(i2c): add polling capabilities to the I2C task

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## Overview

Add the ability to poll the i2c bus, which is used by both the capacitive sensor and pressure sensor.

### CODE REVIEW

File: include/pipettes/core/tasks/i2c_task.hpp [hunk 1]
```diff
@@ -35,6 +35,17 @@ class I2CMessageHandler {
                                    TIMEOUT);
         m.client_callback(m.buffer);
     }
+
+    void visit(PollReadFromI2C &m) {
+        auto empty_array = m.buffer;
+        for (int i = 0; i < m.polling; i++) {
+            i2c_device.central_receive(m.buffer.data(), m.buffer.size(),
```
Comment (sfoster1) [inline include/pipettes/core/tasks/i2c_task.hpp:42]: We should make sure there's a wait or a yield or something somewhere in this loop so we don't busywait the entire cpu. it's also a bit weird to me that the timing isn't defined and isn't saved anywhere, but I don't know what this is for so maybe it's fine.

**Review by sfoster1 — APPROVED** [review]
Functionality looks good but let's make sure we're not eating all the cpu spinning through the poll, and also consider either specifying a frequency to poll or saving the times of the responses or something if this is for data we're going to later do digital signal processing things to

File: include/pipettes/core/tasks/i2c_task.hpp (hunk 1 above)
Comment (Laura-Danielle) [reply include/pipettes/core/tasks/i2c_task.hpp:42]: Yep that was an oversight on my side.

**Review by sfoster1 — APPROVED** [review]
looks good!

**Pull request merged**